docupload/models.py

```diff
@@ -1,13 +1,16 @@
+from django.conf import settings
 from django.db import models
 
 
-# Create your models here.
-
 class Documentation(models.Model):
     '''
     Documentation: Model class which holds details of uploaded documentation file
     '''
 
     name = models.CharField(max_length=200)
-    doc_file = models.FilePathField('../static/docs/')
+    #doc_file = models.FilePathField('../static/docs/')
+    doc_file = models.FileField(upload_to = '%s/media/' %settings.BASE_DIR)
     pub_date = models.DateTimeField('date published')
+
+    def __unicode__(self):
```

Comment:
@Arunabh98 Use `__str__()` instead of `__unicode__()`.... Adhere to python3.5 format